src/server/routes/consents.ts

```diff
@@ -433,7 +433,10 @@ function getABTestPOSTHandler(
 
     try {
       await entitySetter(req.ip, sc_gu_u, req.body);
-      const html = redirectRenderer(url);
+      const html = renderer(url, {
+        pageTitle: PageTitle.RESET_SENT,
+        requestState: state,
+      });
```

Comment:
@coldlink is this right? Be good to get a second pair of eyes on this change

Reply:
In this case, since we've remove the `redirectRenderer` this should become a redirect.

so something like `return res.redirect(303, url)` i think

Reply:
🤦 This makes a lot more sense as a `res.redirect`!